src/pages/index.tsx

```diff
@@ -45,16 +112,254 @@ const Home = () => {
+          />
+          <StepBoxRight
+            title="Caculate a score"
+            img="./images/credit-gauge.svg"
+            description="We run an algorithm on given data to compute a score representing the financial health of a user"
+            style={{ marginBottom: '3rem' }}
+          />
+          <img
+            src="./images/arrow-2.svg"
+            alt="arrow-2"
+            className="absolute top-96 right-5 -z-10 invisible md:visible"
+          />
+          <StepBoxLeft
+            title="Integrate with validators"
+            img="./images/scrtnetwork-logo-white.svg"
+            description="We acquire users financial data by integrating with two validators : Plaid, Coinbase"
+            style={{ marginBottom: '3rem' }}
+          />
+          <img
+            src="./images/arrow-3.svg"
+            alt="arrow-3"
+            className="absolute left-20 invisible md:visible"
+          />
+          <StepBoxRight
+            title="Share your score"
+            img="./images/workicon-combo.svg"
+            description="Make the score available to service providers by releasing permission or providing a public link."
+            style={{ marginBottom: '3rem' }}
+          />
+        </div>
+      ) : (
+        <div className="relative">
+          <StepBoxRight
+            title="Caculate a score"
+            img="./images/credit-gauge.svg"
+            description="We run an algorithm on given data to compute a score representing the financial health of a user"
+            style={{ marginBottom: '3rem' }}
+          />
+          <img
+            src="./images/arrow-2.svg"
+            alt="arrow-2"
+            className="absolute invisible md:visible right-10 top-40"
+          />
+          <StepBoxLeft
+            title="Integrate with validators"
+            img="./images/plaid+coinbase.svg"
+            description="We acquire users financial data by integrating with two validators : Plaid, Coinbase"
+            style={{ marginBottom: '3rem' }}
+          />
+          <img
+            src="./images/arrow-1.svg"
+            alt="arrow-1"
+            className="absolute top-80 -left-20 invisible md:visible"
+          />
+          <StepBoxRight
+            title="Share your score"
+            img="./images/workicon-combo.svg"
+            description="Make the score available to service providers by releasing permission or providing a public link."
+            style={{ marginBottom: '3rem' }}
+          />
+          <img
+            src="./images/arrow-2.svg"
+            alt="arrow-2"
+            className="absolute right-10 invisible md:visible"
+          />
+          <StepBoxLeft
+            title="Integrate with validators"
+            img="./images/scrtnetwork-logo-white.svg"
+            description="We acquire users financial data by integrating with two validators : Plaid, Coinbase"
+            style={{ marginBottom: '3rem' }}
```

Comment:
Can you fix the order here for the provider? 
Providers don't calculate a score nor do they integrate with validators but you can change the wording so that they know that users do that. Also you should mention using the viewing key or query permit provided by the user to get a score.